src/telemetry/filters.ts

```diff
@@ -0,0 +1,74 @@
+import { ITelemetryItem } from '@microsoft/applicationinsights-web';
+import {
+  DEVX_API_URL,
+  GRAPH_API_SANDBOX_URL,
+  GRAPH_URL,
```

Comment:
We also make calls to mgt.dev to the toolkit component. Should we add that here? Do we need to consider sanitization of those requests and responses since they appear to contain user information?

Reply:
We are currently only interested in DevX and Graph APIs. Sanitization is being done for requests to Graph API. There's nothing to sanitize for DevX API calls.

Reply:
Got it, I now see that DevX API calls for snippets and autocomplete send info in the body and that the permissions API calls have a sanitized URL. 

`https://graphexplorerapi.azurewebsites.net/permissions?requesturl=/users/{users-id}/messages&method=GET`